docs/model_library/strategic_water_management/index.rst

```diff
@@ -1451,51 +1463,69 @@ For :math:`t > 1`:
         + \sum_{k \in K | (s, k) \in SKT}\textcolor{purple}{F_{s,k,t}^{Trucked}}
         + \textcolor{purple}{F_{s,t}^{StorageEvaporationStream}})
 
-**Treatment Site Water Quality** :math:`\forall \textcolor{blue}{r \in R}, \textcolor{blue}{w \in W}, \textcolor{blue}{t \in T}`
+**Treatment Feed Water Quality** :math:`\forall \textcolor{blue}{r \in R}, \textcolor{blue}{qc \in QC}, \textcolor{blue}{t \in T}`
 
-The water quality at treatment sites is dependent on the flow rates into the treatment site and the water quality of the flows. Even mixing is assumed, so all outgoing flows have the same water quality. The treatment process does not affect water quality.
+The water quality at treatment sites is dependent on the rate and quality of the feed stream(s) into the treatment site. Even mixing is assumed in calculating the quality of the combined feed stream.
```

Comment:
```suggestion
The water quality at treatment sites is dependent on the flow rates and qualities of the feed streams into the treatment site. Even mixing is assumed in calculating the quality of the combined feed stream.
```
Sorry to do this again - I should have been more careful the first time I rewrote this sentence.